content/administration/maintain/on_premise.rst

```diff
@@ -28,11 +27,11 @@ Solutions
 
 * Do you have a valid Enterprise subscription?
 
-  * Check if your subscription details get the tag "In Progress" on
+  * Check if your subscription details have the tag "In Progress" on
```

Comment:
```suggestion
  * Check if your subscription details have the tag :guilabel:`In Progress` on
```
We use the [guilabel markup](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#gui-element) when we identify a specific UI element.